syntax = "proto3";

package synapse;

enum SpectralFilterMethod {
  kLowPass = 0;
  kHighPass = 1;
  kBandPass = 2;
  kBandStop = 3;
}

message SpectralFilterConfig {
  SpectralFilterMethod method = 1;
  uint32 low_cutoff_mHz = 2;
  uint32 high_cutoff_mHz = 3;
}
